powerhome/playbook
# [PLAY-1388] Online Status Kit Props: Size and No Border

**What does this PR do?**

- ✅ Add a Size prop of "sm", "md", or "lg" with a default of "sm"
- ✅ Add a No Border prop which removes borders
- ✅ Make borders "outer" instead of "inner" (switch from `border-box` to `content-box`)
- ✅ Document these props
- ✅ Set sizes when used with Avatars ('xxs', 'xs' = 'sm'. 'sm', 'md' = 'md'. 'lg', 'xl' = 'lg')
- ✅ Position using global props instead of Sass styles
- ✅ Update Rails Specs and Add Jest Tests

**Screenshots:**
![Zight 2024-08-01 at 11 02 32 AM](https://github.com/user-attachments/assets/81157c5b-9e99-4dc2-8579-bcfec7e1e507)
![Zight 2024-08-07 at 8 36 09 AM](https://github.com/user-attachments/assets/4a11fb73-0bc9-4a59-8c26-1242d7a4fc34)

**How to test?**
1. Go to [/kits/online_status/rails](https://pr3563.playbook.beta.hq.powerapp.cloud/kits/online_status/rails)
2. Review the size and no border examples
3. Turn on dark mode
4. The borders should be dark and blend into the background
5. Go to [/kits/avatar/rails](https://pr3563.playbook.beta.hq.powerapp.cloud/kits/avatar/rails)
6. The online status dot next to the avatar should grow bigger and match the Handoff (review the positioning too)
7. Turn on dark mode and make sure the online status border is dark
8. Repeat the above, but with react.


#### Checklist:
